From b99d31bfbc2cc973355ab5541908f0ee820b24fa Mon Sep 17 00:00:00 2001
From: Portisch <hugo.portisch@yahoo.de>
Date: Fri, 27 Dec 2019 20:11:58 +0100
Subject: [PATCH] multi decoders: limit maximum number of decoder in parallel
 to 1   Kodi can only handle one decoder (12/2019). Because of threading
 issues in Kodi   it is needed to limit the multi decoder instances to a
 maximum of 1.   If Kodi do implement multiple decoder support in parallel
 this decoder limit   needs to be reverted.

---
 drivers/frame_provider/decoder/utils/vdec.h         | 2 +-
 drivers/frame_provider/decoder/utils/vdec_profile.c | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/frame_provider/decoder/utils/vdec.h b/drivers/frame_provider/decoder/utils/vdec.h
index 18d7cd1..56f044a 100644
--- a/drivers/frame_provider/decoder/utils/vdec.h
+++ b/drivers/frame_provider/decoder/utils/vdec.h
@@ -45,7 +45,7 @@ struct device *get_vdec_device(void);
 int vdec_module_init(void);
 void vdec_module_exit(void);
 
-#define MAX_INSTANCE_MUN  9
+#define MAX_INSTANCE_MUN  1
 
 #define VDEC_DEBUG_SUPPORT
 
diff --git a/drivers/frame_provider/decoder/utils/vdec_profile.c b/drivers/frame_provider/decoder/utils/vdec_profile.c
index d00987a..55e5213 100644
--- a/drivers/frame_provider/decoder/utils/vdec_profile.c
+++ b/drivers/frame_provider/decoder/utils/vdec_profile.c
@@ -42,8 +42,6 @@ static uint dec_time_stat_reset;
 
 struct dentry *root, *event;
 
-#define MAX_INSTANCE_MUN  9
-
 struct vdec_profile_time_stat_s {
 	int time_6ms_less_cnt;
 	int time_6_9ms_cnt;
-- 
2.17.1

